README.md

```diff
@@ -9,38 +9,26 @@ This extension adds a "Log in as user" button on user profiles for administrator
 Use [Bazaar](https://discuss.flarum.org/d/5151-flagrow-bazaar-the-extension-marketplace) or install manually:
 
 ```bash
-composer require flagrow/impersonate
+composer require fof/impersonate
 ```
 
 ## Updating
 
 ```bash
-composer update flagrow/impersonate
+composer update fof/impersonate
 php flarum migrate
 php flarum cache:clear
 ```
 
 ## Configuration
 
 You can configure which groups can impersonate users by going to *Permissions > Login as other users*.
-Use with caution, as anybody with that permission can easily access the private data of every user on the forum **and change any admin setting / permission / group by logging in as an administrator !**
-
-## Support our work
-
-We prefer to keep our work available to everyone.
-In order to do so we rely on voluntary contributions on [Patreon](https://www.patreon.com/flagrow).
-
-## Security
-
-If you discover a security vulnerability within Impersonate, please send an email to the Gravure team at security@gravure.io. All security vulnerabilities will be promptly addressed.
-
-Please include as many details as possible. You can use `php flarum info` to get the PHP, Flarum and extension versions installed.
+Use with caution, as anybody with that permission can easily access the private data of every user on the forum **non-admins cannot impersonate an admin account, for security reasons**
 
 ## Links
 
+- [![OpenCollective](https://img.shields.io/badge/donate-friendsofflarum-44AEE5?style=for-the-badge&logo=open-collective)](https://opencollective.com/fof/donate)
 - [Flarum Discuss post](https://discuss.flarum.org/d/9868-flagrow-impersonate-login-as-other-users)
```

Comment:
The discussion slug should be removed just like in composer.json (in particular because it still contains flagrow and also because we usually removed it everywhere nowadays)